packages/web-app-files/src/index.js

```diff
@@ -72,6 +73,16 @@ const navItems = [
       path: `/${appInfo.id}/shares/via-link`
     }
   },
+  {
+    name: $gettext('Spaces'),
+    iconMaterial: 'space',
+    route: {
+      path: `/${appInfo.id}/spaces/projects`
+    },
+    enabled(capabilities) {
+      return capabilities.core.status.productname !== 'ownCloud'
```

Comment:
Please don't introduce checks on the product name. ;-) we need to introduce a capability for spaces in ocis. Could you have a look at e.g. the archiver or app_provider capabilities in ocis and do a similar thing for spaces in a PR to the ocis repo? I'd like to have it versioned, same as with archivers and app_providers.

Reply:
should exist already... maybe you can just use it https://github.com/cs3org/reva/pull/2015